NuGet.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<configuration>
```

Comment:
Can this change not be made to https://github.com/dotnet/project-system/blob/main/build/import/Packages.targets#L5 instead of introducing a NuGet.config file?

Reply:
Yes, that's part of #7191. I'm trying to get rid of this file and the workarounds surrounding ibc. The file at the time existed only in private feeds so our pipelines couldn't capture it and this file contained other feeds that gave access to it. I'll revert this PR back to a draft and fix that before it's merged

Reply:
Fixed in #7233